Development

Kmayankkr opened the pull request.

orensbruli committed:
Beginning the implementation of the node groups
Added new Node groups menu and actions
Implemented convenience method get_selected_nodes == selected_nodes

orensbruli committed:
Coding the node groups methods and actions
Fixed a bug with the context menu of QNetworkxWidget
Disabled the Add to group action on context menu when no node is selected
Implemented the method to add new nodes to a node group
Fixed some problems with non unicode node labels

orensbruli committed:
Added context menu option and methods to change the mass center for the nodes.
Now it can be changed with right click on any place of the scene.

Kmayankkr committed:
Update QNetworkxGraph.py

orensbruli committed:
Merge pull request #32 from Kmayankkr/development

Bug fix for nested context menus

Kmayankkr committed:
Update QNetworkxGraph.py

Kmayankkr committed:
Merge pull request #33 from Kmayankkr/development

Update QNetworkxGraph.py

orensbruli committed:
Implemented the set_mass_center for groups of nodes.
set_mass_center is applied to the selected nodes or all if none is selected
Fixed bug to ignore avoid deselection of nodes when right click on background
Fix for QString to unicode

Kmayankkr committed:
Update QNetworkxGraph.py

Kmayankkr committed:
Merge pull request #35 from Kmayankkr/development

Update QNetworkxGraph.py

Kmayankkr committed:
Update QNetworkxGraph.py

Kmayankkr committed:
Merge branch 'development' into development

Kmayankkr committed:
Update QNetworkxConfig.py

Kmayankkr committed:
Update QNetworkxGraph.py

Kmayankkr committed:
Update QNetworkxGraph.py

Kmayankkr committed:
Merge pull request #36 from Kmayankkr/development

Update QNetworkxGraph.py

Kmayankkr committed:
Update QNetworkxConfig.py

Kmayankkr committed:
Update QNetworkxGraph.py

Kmayankkr committed:
Merge pull request #37 from Kmayankkr/development

Refactored self.isComponentRunning

orensbruli committed:
Changed the Styles system to be changed from json files.

orensbruli committed:
Fixed a default path problem on the for the styles directory.

Kmayankkr committed:
Update QNetworkxGraph.py

Kmayankkr committed:
Update QNetworkxGraph.py

Kmayankkr committed:
Merge pull request #39 from Kmayankkr/development

Update QNetworkxGraph.py

orensbruli commented:
@Kmayankkr Have you executed or made some test of the development branch before requesting the merge? Do you think it's safe to do? I have been some time without using the library so I trustwhat you think...

orensbruli committed:
Added code for pysyde2

orensbruli committed:
Added convenience method clear

orensbruli committed:
Fixed bug it graph is empty.

orensbruli committed:
Merge branch 'development_pyside2' into development

orensbruli committed:
Added constructor option for QNetworkxControler to assign an external view_widget,
